app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="row inset">
+<div class="row inset" style="margin-top:60px">
```

Comment:
avoid inline styles

Reply:
done, added style to layout.scss and updated file